doc/source/timeseries.rst

```diff
@@ -889,9 +889,31 @@ The key features of a ``DateOffset`` object are:
 * It can be multiplied by an integer (positive or negative) so that the
   increment will be applied multiple times.
 * It has :meth:`~pandas.DateOffset.rollforward` and
-  :meth:`~pandas.DateOffset.rollback` methods for moving a date forward or 
+  :meth:`~pandas.DateOffset.rollback` methods for moving a date forward or
   backward to the next or previous "offset date".
 
+Beware that adding an offset to a ``DatetimeIndexes`` with consecutive values
```

Comment:
I'd be useful to wrap this entire section with a `.. note::` directive so it's stands out as a caveat when rendering.